# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1295580894 28800
#      Thu Jan 20 19:34:54 2011 -0800
# Node ID ab61de78833ee4575676f6a36661838760ae35d7
# Parent  1d13679b587e54ccad97d832bf7cdba2ffc3ec35
Update README for documentation scripts.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2011-01-20  Rik  <octave@nomad.inbox5.com>
+
+	* doc/interpreter/doccheck/README: Update documentation for
+	documentation scripts.
+
 2011-01-20  Rik  <octave@nomad.inbox5.com>
 
 	* doc/interpreter/doccheck/aspell-octave.en.pws,
 	doc/interpreter/plot.txi: Use @nospell macro on certain words in
 	docstring.
 
 2011-01-20  John W. Eaton  <jwe@octave.org>
 
